exercises/practice/clock/.meta/config.json

```diff
@@ -36,5 +36,8 @@
     ]
   },
   "source": "Pairing session with Erin Drummond",
-  "source_url": "https://twitter.com/ebdrummond"
+  "source_url": "https://twitter.com/ebdrummond",
+  "custom": {
+    "allowed-to-not-compile": "Stub doesn't compile because there is no to_string() implementation.  This exercise is an introduction to derived and self-implemented traits,  therefore adding template for a trait would reduce student learning."
```

Comment:
```suggestion
    "allowed-to-not-compile": "Stub doesn't compile because there is no to_string() implementation. This exercise is an introduction to derived and self-implemented traits, therefore adding template for a trait would reduce student learning."
```